java/vector/src/main/java/org/apache/arrow/vector/BitVector.java

```diff
@@ -504,6 +504,8 @@ public void setRangeToOne(int firstBitIndex, int count) {
         validityBuffer.setByte(i, 0xFF);
         valueBuffer.setByte(i, 0xFF);
       }
+      validityBuffer.setOne(startByteIndex, endByteIndex - startByteIndex);
```

Comment:
shouldn't some code be remove here?

Reply:
Sorry for my mistake. Redundant code has been removed.